examples/step-90/step-90.cc

```diff
@@ -0,0 +1,1344 @@
+    TimerOutput        computing_timer;
+
+    // However, the degrees of their FE spaces may be different
+    const unsigned int fe_degree;
+    const unsigned int level_set_fe_degree;
+
+    // The first two quadratures are for the bulk and face integration, typical
+    // for TraceFEM stabilizations, while the integration over implicit surface
+    // is based on the last, one-dimensional rule.
+    const QGauss<dim>     cell_quadrature;
+    const QGauss<dim - 1> face_quadrature;
+    const QGauss<1>       quadrature_1D;
+
+    // We need three separate FE spaces.
+    // The first manages the TraceFEM space which is active on intersected
+    // element. The second manages the discrete
+    // level set function that describes the geometry of the surface. The third
+    // handles a piecewise constant space which is used solely for the
+    // accumulation of refinement indicators and of errors.
+    hp::FECollection<dim> fe_collection;
+    const FE_Q<dim>       level_set_fe;
+
+    // The corresponding DOF-handlers are given by
+    DoFHandler<dim> dof_handler;
+    DoFHandler<dim> level_set_dof_handler;
+
+    // Since we will adaptively refine the bulk triangulation, two constraints
+    // are needed: one for the solution space and another for the level-set
+    // space.
+    AffineConstraints<double> constraints;
+    AffineConstraints<double> level_set_constraints;
+
+    // Discrete vectors initialized with dof_handler and level_set_dof_handler.
+    VectorType    completely_distributed_solution;
+    VectorType    locally_relevant_solution;
+    VectorType    locally_relevant_exact;
+    VectorType    level_set;
+    Vector<float> activeFE_indicator;
+
+    // Mesh_classifier separates intersected elements and non-intersected ones
+    // in the fe_collection
+    NonMatching::MeshClassifier<dim> mesh_classifier;
+    const MappingQ1<dim>             mapping;
+
+    // Any TraceFEM need a stabilization, and we choose the normal-gradient,
+    // volume stabilization.
+    NormalGradientVolumeStabilization<dim> stabilization_scheme;
+
+    // Discrete right-hand side and the final matrix correspondent to
+    // dof_handler.
+    VectorType global_rhs;
+    MatrixType global_matrix;
+
+    SparsityPattern sparsity_pattern;
+
+    IndexSet locally_owned_dofs;
+    IndexSet locally_relevant_dofs;
+
+    // Depending on the type of the quadrature, surface, face or volume, we need
+    // to define different update flags.
+    NonMatching::RegionUpdateFlags surface_update_flags;
+    UpdateFlags                    face_update_flags;
+    UpdateFlags                    cell_update_flags;
+
+    // The following displays the errors of the convergence test.
+    ConvergenceTable convergence_table;
+    unsigned int     number_of_iterations;
+    double           average;
+    double           area;
+    double           error_L2_sqr;
+    double           error_H1_sqr;
+    double           error_stab_sqr;
+
+    int number_of_proc;
```

Comment:
This should probably be an `unsigned int`. Separately, out usual convention is to call variables that store the number of something as `n_something`. Same for `number_of_iterations` above.